TypeCobol.Test/Parser/Programs/Cobol85/ContinuationLine/ContinuationPictureString.rdzMix.txt

```diff
@@ -0,0 +1,14 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. DVZZMFT0.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 STR1.
+          05 data1                                              PIC    X
+      -    X.
+       01 STR2.
```

Comment:
I don't see why this second picture test.

Reply:
Originally, the behavior was  different on these two different lines so I used directly the source code from the issue in the test.

Reply:
Maybe if the behavior is now the same we can delete thes second test.

Reply:
We should check how the parser reacts with
```cob
       01 STR2.
           05 data2                                        Pic 9(4) comp
      -    -3.
```

Reply:
See d922866c0f084e1fe573589d3089411bdaaa1b7b.